### ISSUE TITLE
Buttons don't work sometimes.

### ISSUE THREAD

**Issue by ssss141414** [microsoft/BotFramework-Composer#2418]
## Describe the bug
Some times the buttons don't work.
![image](https://user-images.githubusercontent.com/18118994/77872312-38562380-7279-11ea-93df-ce1265251f69.png)
I can't reproduce, but it mostly happened when I have made a lot of operation like cut, copy, paste from these dialog to those, etc...
Nothing happens when you click Copy/Cut/Undo/Redo. Paste will paste a old one that you ever copyed(because Copy doesn't work now).

## Browser

- [x] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

**Comment by boydc2014** [comment]
@ssss141414 can you provide a simpler approach to repro?

**Comment by KayMKM** [comment]
I met this bug too. But not sure how to repro it...

**Comment by yeze322** [comment]
Had a live debug with Shiyi @ssss141414 , I found this bug is related to iframe and ApiClient. A `Delete` event was fired but there was no listener in the ApiClient.

Considering that we have a plan to remove iframe wrapper #2121 , the ApiClient should also be retired along with iframe, so I won't spend more time on digging why ApiClient failed to receive a message. @boydc2014 

@KayMKM After we closed #2121 , this bug should be resolved as well.
